.github/workflows/nightly.yml

```diff
@@ -9,6 +9,8 @@ jobs:
   create-nightly-tag:
     runs-on: ubuntu-latest
 
+    if: github.repository == 'streamlit/streamlit'
+
```

Comment:
Currently, nightly running on forks - adding conditional to only run on streamlit/streamlit repo (all following jobs dependent on create-nightly-tag running successfully, so they will be skipped if  create-nightly-tag is skipped)